deploy/services-demo/conf/nginz/nginx.conf

```diff
@@ -143,7 +143,12 @@ http {
       proxy_pass http://brig;
     }
 
-    location /brig/api-docs {
+    location /api/swagger-ui {
+        include common_response_no_zauth.conf;
+        proxy_pass http://brig;
+    }
+
+    location /api/swagger.json {
```

Comment:
I think we'll also need to change the HTML file in the old swagger-ui to match this change.

Reply:
I removed those entries now.